kyo-core/shared/src/main/scala/kyo/stats/attributes.scala

```diff
@@ -3,14 +3,14 @@ package kyo.stats
 import kyo.stats.Attributes.AsAttribute
 import scala.annotation.implicitNotFound
 
-case class Attributes(get: List[Attributes.Attribute]) extends AnyVal {
```

Comment:
Used only for traces now, so I moved it to from `kyo-scheduler` to `kyo-core`, which uses Scala 3 syntax.